source/extensions/filters/listener/tls_inspector/tls_inspector.cc

```diff
@@ -115,19 +131,28 @@ void Filter::onALPN(const unsigned char* data, unsigned int len) {
     }
     protocols.emplace_back(reinterpret_cast<const char*>(CBS_data(&name)), CBS_len(&name));
   }
-  cb_->socket().setRequestedApplicationProtocols(protocols);
-  alpn_found_ = true;
+  onAlpnCb(protocols);
+  alpn_found = true;
+}
+
+void Filter::onServername(absl::string_view servername) {
+  ENVOY_LOG(debug, "tls:onServerName(), requestedServerName: {}", servername);
```

Comment:
Should we keep this similar debug log in [tcp_proxy](https://github.com/envoyproxy/envoy/blob/master/source/common/tcp_proxy/tcp_proxy.cc#L472-L473)

Reply:
I think it can be useful for debug to track that the SNI in both filters